# Exploring Pharmacogenomic LOD for Molecular Explanations of Gene-Drug Relationships

### Representative: Adrien Coulet (coulet -- @ -- loria.fr)

## Gitter room
BioHackathon 2018 Paris/07-PharmaLOD

## Hacking task presentation
https://github.com/elixir-europe/BioHackathon/blob/master/interoperability/Exploring%20Pharmacogenomic%20LOD%20for%20Molecular%20Explanations%20of%20Gene-Drug%20Relationships/task7_adrien_coulet.pdf

## Length
Through all Biohackathon week (12-16 Nov.)

## Community
---

Collaborators of the ANR PractiKPharma project -- http://practikpharma.loria.fr/

## Leads
---
- Adrien Coulet (all week)
- Clément Jonquet (Nov. 12-14)
- Bastien Rance (on Skype)
- Bousquet Cedric (all week)

## Background information
---
The idea of the hacking task is to find elements of mechanistic/molecular explanations of pharmacogenomic (PGx) drug-gene relationships present in our PGx Linked Data set, named PGxLOD (https://pgxlod.loria.fr/). 
Proposing molecular explanation can be seen as predicting novel links between a drug and a target protein, or between a drug and a gene to a same unique pathway. 
We consider these links as interpretation elements to experts who wonder why a gene impacts a drug response. 
This may include enriching of our initial linked data, adding provenance elements and experimenting link prediction methods (or others).

## Expected outcomes
---

1/ Enrichment and reuse of our Pharmacogenomic LOD
2/ Getting our Pharmacogenomic LOD more FAIR
3/ Learning from the Elixir network experience
4/ Developing network for future projects at the interface between bioinformatics and medical informatics
5/ Networking for our PhD students and postdocs

## Expected audience
---

Python programmers, RDF fans, FAIR evangelists, AI enthousiasts


## Related works and references
---

Presentation of the PractiKPharma project: https://ercim-news.ercim.eu/en104/special/mining-electronic-health-records-to-validate-knowledge-in-pharmacogenomics

Basic schema for our pharmacogneomics LOD: https://peerj.com/preprints/3140/

Article about knowledge comparison within PGxLOD: https://www.biorxiv.org/content/early/2018/11/07/390971

## References
---
Previous work on link prediction from the LOD (Many thanks to Maulik Kamdar for these links):

  Dalleau et al.: https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-017-0125-1

  Maulik R. Kamdar, Mark A. Musen:
  PhLeGrA: Graph Analytics in Pharmacology over the Web of Life Sciences Linked Open Data. WWW 2017: 321-329
  Mining the Web of Life Sciences Linked Open Data for Mechanism-Based Pharmacovigilance. WWW (Companion Volume) 2018: 861-865
  
  Munoz et al.: Facilitating prediction of adverse drug reactions by using knowledge graphs and multi-label learning models.   
  https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx099/4085292

  The KB workgroup of OHDSI: Large-scale adverse effects related to treatment evidence standardization (LAERTES): an open scalable system 
  for linking pharmacovigilance evidence sources with clinical data
  https://link.springer.com/article/10.1186/s13326-017-0115-3
  
  Ayvaz et al.:Toward a complete dataset of drug–drug interaction information from publicly available sources 
  https://www.sciencedirect.com/science/article/pii/S1532046415000738

  Noor et al.: Drug-drug interaction discovery and demystification using Semantic Web technologies
  https://academic.oup.com/jamia/article-lookup/doi/10.1093/jamia/ocw128
  
  Malec et al.: Using the Literature to Construct Causal Models for Pharmacovigilance
  https://easychair.org/publications/preprint/X6kk

## GitHub or any other public repositories of your FOSS products (if any)
---

- [Repositories for the Pharmacogenomic LOD developed at LORIA (PGxLOD)](https://gitlab.inria.fr/pgxlod/)
- [Repository of French biomedical ontologies](http://bioportal.lirmm.fr/)

## Hackers
---

Adrien Coulet
Andon Tchechmedjiev
Cedric Bousquet
Francois-Elie Calvier
William Digan
Joel Legrand
Clement Jonquet
Pierre Monnin
Malika Smail Tabbone (remote)
Bastien Rance (remote)
Athenais Vaginay (remote)
[to complete]


